ray-operator/config/manager/manager.yaml

```diff
@@ -26,7 +26,8 @@ spec:
       containers:
       - command:
         - /manager
-#        args:
+        args:
+        - --feature-gates=RayClusterStatusConditions=true
```

Comment:
We need this to make [buildkite/ray-ecosystem-ci-kuberay-ci/test-sample-yamls-nightly-operator](https://buildkite.com/ray-project/ray-ecosystem-ci-kuberay-ci/builds/5712#0193d390-b9f9-42ec-9c81-1754786d87a1), where the feature gate is not enabled by default, to pass.

Reply:
Isn't that using nightly build though? Nightly build should have this feature gate enabled by default at this point?

Reply:
Oh, I copied the wrong link. I should mean this one [buildkite/ray-ecosystem-ci-kuberay-ci/test-sample-yamls-latest-release](https://buildkite.com/ray-project/ray-ecosystem-ci-kuberay-ci/builds/5712#0193d390-b9fa-4fad-9911-cbf9298d1794)

Reply:
> ah I see, makes sense. Can you add a comment that we can remove this once v1.3 is released?

Yes. The comment is added.